### ISSUE TITLE
Crashes on startup using MultiMC (1.83)

### ISSUE THREAD

**Issue by Elinvynia** [NillerMedDild/Enigmatica2Expert#1970]
### Modpack Version

1.83

### Describe your issue.

Crashes on startup with obscure Java errors.

Linux, Fedora 35, GNOME, Wayland, proprietary NVIDIA drivers

Java is version 1.8.0_312, 64-bit

Microsoft account

Other modpacks work fine using MultiMC

### Crash Report

https://pastebin.com/JsAFVb5u

### Latest Log

https://pastebin.com/08ziDhK0

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Both

**Comment by Elinvynia** [comment]
Possibly relevant line from MultiMC log itself:

```
11.257 C BAD stuff happened to mod json:
   11.258 C "{
  "modListVersion": "2",
  "modList":
  [
    {
      "modid": "openblocks",
      "name": "OpenBlocks",
      "description": "All the things you never thought you will ever need",
      "version": "1.8.1",
      "mcversion": "1.12.2",
      "url": "http://openmods.info/",
      "updateUrl": "",
      "authorList": [
        "Mikee",
        "NeverCast",
        "boq",
        "Lyqyd"
      ],
      "credits": "",
      "logoFile": "",
      "screenshots": [
      ],
      "parent": ""
    }
  ]
}
"
```

**Comment by NillerMedDild** [comment]
Thanks for the report... It's super weird 😄 have you tried re-downloading / importing into MultiMC?

**Comment by Elinvynia** [comment]
Tried clean install of both stable and development versions of MultiMC, tried re-downloading e2e multiple times, unfortunately, the same error still persists.

Also happpens with 1.82a and 1.82 (didn't go further).

Just for the sake of it I also tried the skyblock version (1.36) and that gave the same error.

Meanwhile E6 and all other modpacks I have tried so far work perfectly well.

I remember playing E2E half a year ago perfectly well, which is weird since that was the 1.82 version which now doesn't work anymore.

I'll try GDLauncher just to be sure it's not a MultiMC error.
